src/hotspot/share/opto/superword.hpp

```diff
@@ -531,9 +545,9 @@ class SuperWord : public ResourceObj {
   // Create a vector operand for the nodes in pack p for operand: in(opd_idx)
   Node* vector_opd(Node_List* p, int opd_idx);
   // Can code be generated for pack p?
-  bool implemented(Node_List* p);
+  bool implemented(const Node_List* p);
```

Comment:
Could be made `const` as well if you also make `is_marked_reduction()` const.

Reply:
I could. But I don't want to push this too far in this RFE. I could also make the whole function const.
Maybe I can have a future RFE where I just make everything const that can be const in SuperWord?
But also once I refactor out all the components (e.g. Reductions), then I can just make things const at that point.

Reply:
Up to you, I just thought since you changed the argument to `const` that you could also make the entire method `const`. But could also be done later.